lib/px/pxnginx.lua

```diff
@@ -175,7 +175,7 @@ function M.application(px_configuration_table)
     end
 
     -- handle pxvid cookie
-    local pxvid = ngx.var.cookie__pxvid
+    local pxvid = ngx.var.cookie__pxvid or ngx.var.cookie_pxvid
```

Comment:
missed that on the original pr, but does this syntax work in lua?

Reply:
yap.. or is a valid test